workspaces/ui-v2/config/paths.js

```diff
@@ -50,14 +50,19 @@ const resolveModule = (resolveFn, filePath) => {
   return resolveFn(`${filePath}.js`);
 };
 
+let entrypoints = {
+  cloud: "src/entry-points/cloud/index",
+  local: "src/entry-points/local/index"
+}
+
 // config after eject: we're in ./config/
 module.exports = {
   dotenv: resolveApp('.env'),
   appPath: resolveApp('.'),
   appBuild: resolveApp(buildPath),
   appPublic: resolveApp('public'),
   appHtml: resolveApp('public/index.html'),
-  appIndexJs: resolveModule(resolveApp, 'src/entry-points/local/index'), //@TODO: parameterize this
+  appIndexJs: resolveModule(resolveApp, entrypoints[process.env.ENTRYPOINT || "local"]),
```

Comment:
I would namespace the env with OPTIC__ => OPTIC__WEBPACK_ENTRY_POINT or something like that

Reply:
Done